src/linux/epoll.rs

```diff
@@ -295,20 +295,15 @@ impl Epoll {
     /// let ev_count = epoll.wait(10, -1, &mut ready_events[..]).unwrap();
     /// assert_eq!(ev_count, 1);
     /// ```
-    pub fn wait(
-        &self,
-        max_events: usize,
-        timeout: i32,
-        events: &mut [EpollEvent],
-    ) -> io::Result<usize> {
+    pub fn wait(&self, timeout: i32, events: &mut [EpollEvent]) -> io::Result<usize> {
```

Comment:
`max_events` should be removed from the arguments list as well: https://github.com/rust-vmm/vmm-sys-util/blob/master/src/linux/epoll.rs#L268. Also the documentation example and the unit tests should be updated.

Reply:
Cool, doing that now. My local won't pass some unrelated unit tests, so sorry for pushing so many times haha. Just letting your CI take care of it. Unit tests may not work on M1 Mac Mini:

```
test rand::tests::test_timestamp_cycles ... FAILED
test rand::tests::test_xor_psuedo_rng_u32 ... FAILED
test unix::tempdir::tests::test_create_dir ... FAILED
test unix::tempdir::tests::test_create_dir_in ... FAILED
```

I'll open a different issue around this.

Reply:
No problem, would be nice if you can also add a CHANGELOG entry in the `Unreleased` section. Thanks!

Reply:
Done!